src/GitHub.TeamFoundation.14/Home/PulseNavigationItem.cs

```diff
@@ -14,7 +14,7 @@ namespace GitHub.VisualStudio.TeamExplorer.Home
     [PartCreationPolicy(CreationPolicy.NonShared)]
     public class PulseNavigationItem : TeamExplorerNavigationItemBase
     {
-        public const string PulseNavigationItemId = "5245767A-B657-4F8E-BFEE-F04159F1DDA2";
+        public const string PulseNavigationItemId = "5245767A-B657-4F8E-BFEE-F04159F1DDA5";
```

Comment:
I'm curious why the Guid changed for `PulseNavigationItem`?

Reply:
Reverted